pkg/policy/groups/actions.go

```
@@ -162,13 +166,15 @@ func addDerivativeCNP(cnp *cilium_v2.CiliumNetworkPolicy) error {
 	if err != nil {
 		statusErr := updateDerivativeStatus(cnp, derivativeCNP.ObjectMeta.Name, err)
 		if statusErr != nil {
+			metrics.PolicyImportErrors.Inc()
```

In this case, the error is returned to the caller so I think it should already be measured?

Oh, but sometimes the caller ignores the error and in general it tends to be handled by controllers. Hmm, I see. In that case it's probably more consistent to keep this here so at least this entire function guarantees that the error is incremented in bad conditions.

---

Yes, I think we should keep it here. Otherwise we would need to have logic for detecting specific errors in `UpdateController` method of Manager.